x-pack/test/alerting_api_integration/spaces_only/tests/actions/check_registered_connector_types.ts

```diff
@@ -42,6 +42,7 @@ export default function createRegisteredConnectorTypeTests({ getService }: FtrPr
         '.jira',
         '.resilient',
         '.teams',
+        '.tines',
```

Comment:
I see your test is failing because the array is in the wrong order. i think you should add a `.sort()` to these arrays so the order does not matter